apps/studio/components/interfaces/RoleImpersonationSelector/UserImpersonationSelector.tsx

```diff
@@ -95,7 +95,7 @@ const UserImpersonationSelector = () => {
               )
             }
             placeholder="Search for a user.."
-            onChange={(e) => setSearchText(e.target.value.trim())}
+            onChange={(e) => setSearchText(e.target.value.trim().toLocaleLowerCase())}
```

Comment:
for this one actually we shouldn't be doing trim and lowerCase in onChange setSearchText, as that means that the user can enter "space" in the input, nor enter uppercase letters

for this one specifically, we should shift the trim and toLowerCase to L35 here probably
https://github.com/supabase/supabase/blob/master/apps/studio/components/interfaces/RoleImpersonationSelector/UserImpersonationSelector.tsx#L35
![image](https://github.com/user-attachments/assets/9f43c4c6-dc33-44eb-9948-c6788ddf52cf)